### DISCUSSION TITLE
Microsoft Auth - Possible to change UI to display username rather than ID?

### DISCUSSION THREAD

**Discussion by username-is-already-taken2** [concourse/concourse#7541] [Help & Support]
Hi there
When I logon to the concourse UI after authenticating to Azure AD the username in top corner of the UI is the guid for my account 
![image](https://user-images.githubusercontent.com/13700297/133070790-011abf0c-9297-48aa-90ba-59515c2ebd84.png)
Anyone know if I can change this to be my account username or email?  I see those properties being returned in the logs along with the groups I'm a member of.

Thanks in advance

Gary

**Comment by evanchaoli** [comment]
It's configuration since 7.0.0. Refer to https://github.com/concourse/concourse/releases/tag/v7.0.0, search for "Expose username of who manually triggered build to build metadata" in the release note.

**Reply by stiliajohny** [reply]
hi @evanchaoli , is this just for the logs or will it also show the email/username instead of the userID when a user logs in ? 

on my side, my objective is: when a user login to see the name instead of the AuthID, yet your advice is also helpful for me !! 

thanks a lot
